crates/revm/src/handler/handle_types/execution.rs

```diff
@@ -83,13 +83,8 @@ impl<'a, EXT, DB: Database> ExecutionHandler<'a, EXT, DB> {
 
     /// Call frame call handler.
     #[inline]
-    pub fn call(
-        &self,
-        context: &mut Context<EXT, DB>,
-        inputs: Box<CallInputs>,
-        return_memory_offset: Range<usize>,
-    ) -> FrameOrResult {
-        (self.call)(context, inputs, return_memory_offset)
+    pub fn call(&self, context: &mut Context<EXT, DB>, inputs: Box<CallInputs>) -> FrameOrResult {
+        (self.call)(context, inputs.clone(), inputs.return_memory_offset)
```

Comment:
@DoTheBestToGetTheBest same here, remove `inputs.return_memory_offset`

and Check Inspector trait i think same is applicable there

Reply:
> @DoTheBestToGetTheBest same here, remove `inputs.return_memory_offset`
> 
> and Check Inspector trait i think same is applicable there

Hey thanks for your review, does it look now or i should also take off other return_memoy_offset?

Reply:
Docs are failling

Reply:
> > @DoTheBestToGetTheBest same here, remove `inputs.return_memory_offset`
> > and Check Inspector trait i think same is applicable there
> 
> Hey thanks for your review, does it look now or i should also take off other return_memoy_offset?

i think i also have to take off return memory range from make_call_frame

Reply:
@rakita does it look to you better?

Reply:
It does! will review in a few days.